### ISSUE TITLE
Adapt to claim $ARBITRUM airdrop 

### ISSUE THREAD

**Issue by abadon666** [script-money/ens-claim-flashbot-example#28]
https://docs.arbitrum.foundation/

Hello,
Can you adapt scrypt for the next airdrop for ARBITRUM...
https://arbitrum.foundation/
THANKS
My account is eleggible but is compromissed

**Comment by script-money** [comment]
> 

mainnet only

**Comment by abadon666** [comment]
yes Arbitrum airdrop is for mainnet....can you adapt before claim? claim start in 6 days..thanks

**Comment by script-money** [comment]
> yes Arbitrum airdrop is for mainnet....can you adapt before claim? claim start in 6 days..thanks

of course

**Comment by yanrising** [comment]
https://arbiscan.io/address/0x67a24ce4321ab3af51c2d0a4801c3e111d88c9d9
Arbitrum Foundation: Token Distributor

**Comment by script-money** [comment]
> https://arbiscan.io/address/0x67a24ce4321ab3af51c2d0a4801c3e111d88c9d9 Arbitrum Foundation: Token Distributor

It's on Arbitrum, can't use this flashbot tool

**Comment by abadon666** [comment]
is there any solution for Arbitrum chain??

**Comment by yanrising** [comment]
Like this? https://github.com/MrFatoni/arb-claim

**Comment by script-money** [comment]
> 

This approach is similar to that of a hacker swapper, and you can directly use his code.

**Comment by 0xWhale** [comment]
> > This approach is similar to that of a hacker swapper, and you can directly use his code.
> 
> https://bscscan.com/address/0x320F08A6638f27C3e514A37Bc9D0439A580027A4
> 
> Hacker did send fund + claim + move, all in the same block. How possible!!!
> 
> ![image](https://user-images.githubusercontent.com/31094102/226900593-703e12f7-36cb-4b55-a915-aeed3e2e62b7.png)

https://github.com/bnb-chain/bsc/issues/1101

**Comment by tranhoaison** [comment]
> > > This approach is similar to that of a hacker swapper, and you can directly use his code.
> > 
> > 
> > https://bscscan.com/address/0x320F08A6638f27C3e514A37Bc9D0439A580027A4
> > Hacker did send fund + claim + move, all in the same block. How possible!!!
> > ![image](https://user-images.githubusercontent.com/31094102/226900593-703e12f7-36cb-4b55-a915-aeed3e2e62b7.png)
> 
> [bnb-chain/bsc#1101](https://github.com/bnb-chain/bsc/issues/1101)

can I do the same on ARB?

**Comment by 0xWhale** [comment]
> can I do the same on ARB?

There is no such possibility

**Comment by tranhoaison** [comment]
> There is no such possibility

:( my wallet got hacked and it is in hacker sweep-bot. do you have any suggestions to claim and move my fund?
